src/refactor_nrepl/find/find_symbol.clj

```diff
@@ -58,8 +58,7 @@
         (map (juxt (comp :line :env)
                    (comp :end-line :env)
                    (comp :column :env)
-                   (comp :end-column :env)
-                   pred))
```

Comment:
having a `pred` here instructs `juxt` to call 5 functions on a mapped values (producing 5 results), however `zipmap` below goes with 4 elements: `:line-beg :line-end :col-beg :col-end` which, as I understand, ignores last "juxted" value produced by `pred`?